### PR TITLE
feat(mojaloop/#3618): add support for POST /fxQuotes and /fxTransfers

### PR DESCRIPTION
**Pull request by geka-evk**
feat(mojaloop/#3618): add support for POST /fxQuotes and /fxTransfers - https://github.com/mojaloop/project/issues/3618

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,16 +40,16 @@
     }
   ],
   "dependencies": {
+    "@mojaloop/api-snippets": "^17.3.0-snapshot.0",
```
Comment (kleyow) [inline package.json:43]: Typically we don't check in snapshots, whats the blocker on those changes getting merged?

Everything else lgtm.

**Comment by kleyow** [comment]
Functionalliy, looks good. Can you write some unit tests for your functions?

File: package.json (hunk 1 above)
Comment (vijayg10) [reply package.json:43]: Thanks @kleyow . Yes we are merging to feature branch. And the PRs on api-snippets, just got an update from Sam saying he is fine with the spec and we will merge them soon.

File: src/lib/ilp/index.js [hunk 2]
```diff
@@ -35,14 +35,7 @@ class Ilp {
...
      */
-    getResponseIlp(transactionObject) {
-        const ilpData = Buffer.from(base64url(JSON.stringify(transactionObject)));
-        const packetInput = {
-            amount: this._getIlpCurrencyAmount(transactionObject.amount), // unsigned 64bit integer as a string
-            account: this._getIlpAddress(transactionObject.payee), // ilp address
-            data: ilpData // base64url encoded attached data
-        };
-
+    getResponseIlp(transactionObject, packetInput) {
```
Comment (vijayg10) [inline src/lib/ilp/index.js:38]: Shall we rename packetInput to ilpPacketObject or something?

**Pull request merged**